src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random rand = new Random();
+        int ballNum = rand.nextInt(100);
```

Comment:
and replace 100 with constant 
```suggestion
        int ballNumber = rand.nextInt(100);
```